src/mumble/PulseAudio.cpp

```diff
@@ -153,6 +153,28 @@ PulseAudioSystem::~PulseAudioSystem() {
 	pa_threaded_mainloop_free(pam);
 }
 
+QString PulseAudioSystem::outputDevice() const {
+	QString odev = g.s.qsPulseAudioOutput;
+	if (odev.isEmpty()) {
+		odev = qsDefaultOutput;
+	}
+	if (!qhOutput.contains(odev)) {
```

Comment:
I assume this actually needs contains and not equals? In what way will they differ yet still match?